bundesliga_app/views.py

```diff
@@ -141,10 +129,16 @@ class EventDiscountsView(TemplateView, LoginRequiredMixin):
 
     def get_context_data(self, **kwargs):
         context = super(EventDiscountsView, self).get_context_data(**kwargs)
+        # Get event by id in kwargs
         context['event'] = get_object_or_404(
             Event,
-            event_id=self.kwargs['event_id'],
+            id=self.kwargs['event_id'],
```

Comment:
I think it should be more describable of which id is..

Reply:
Its a get, so Its not a variable, its an attribute of the Event model (the primary key in this case)